homeassistant/components/circuit/__init__.py

```diff
@@ -0,0 +1,50 @@
+"""The Unify Circuit component."""
+
+import logging
+
+import voluptuous as vol
+
+from homeassistant.const import CONF_NAME, CONF_URL
+from homeassistant.helpers import config_validation as cv, discovery
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = "circuit"
+CONF_WEBHOOK = "webhook"
+
+WEBHOOK_SCHEMA = vol.Schema(
+    {
+        vol.Optional(CONF_NAME): cv.string,
+        vol.Required(CONF_URL): vol.All(cv.ensure_list, [cv.string]),
+    }
+)
+
+CONFIG_SCHEMA = vol.Schema(
+    {
+        DOMAIN: [
```

Comment:
Please remove the list here.

Reply:
list removed, this gives a configuration.yaml change too:
```
circuit:
  webhook:
    - name: circuit_webhook_test
      url: https://eu.yourcircuit.com/rest/v2/webhooks/incoming/<ID-1>
    - name: circuit_hass_dev
      url: https://eu.yourcircuit.com/rest/v2/webhooks/incoming/<ID-2>
    - name: circuit_Webhook_group
      url:
        - https://eu.yourcircuit.com/rest/v2/webhooks/incoming/<ID-1>
        - https://eu.yourcircuit.com/rest/v2/webhooks/incoming/<ID-2>
```